src/components/LoginForm/LoginForm.tsx

```diff
@@ -1,37 +1,40 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Formik } from 'formik';
 import { useNavigate } from 'react-router-dom';
-import './LoginForm.css';
+import useAuth from '../../contexts/useAuth';
 import Logo from '../Common/Logo';
 import Input from '../Common/Input';
 import Button from '../Common/Button';
 import Link from '../Common/Link';
+import './LoginForm.css';
 
 import { loginValidator } from './loginFormValidator';
 
 export interface LoginFormProps {
   customClass?: string;
```

Comment:
why do you need to use this property? Will You use the same component for other features later?

Reply:
You are right, it's not necessary anymore